Portals/_default/Skins/Vanjaro/Base.ascx.cs

```diff
@@ -125,7 +125,8 @@ protected void Page_Load(object sender, EventArgs e)
             JavaScript.RequestRegistration(CommonJs.jQuery, null, SpecificVersion.Latest);
 
             string ThemeJS = "~/Portals/_default/vThemes/" + Core.Managers.ThemeManager.CurrentTheme.Name + "/theme.js";
-            if (File.Exists(System.Web.Hosting.HostingEnvironment.MapPath(ThemeJS)))
+            
+            if (Core.Managers.ThemeManager.HasScript(ThemeJS))
```

Comment:
Pls refactor. There's no need to pass ThemeJS; ThemeManager can generate that based on the CurrentTheme.Name